airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e55879a8-0ef8-4557-abcf-ab34c53ec460
-  dockerImageTag: 1.5.0
+  dockerImageTag: 1.5.1
```

Comment:
Can you bump also the doc changelog?
After that I think it's fine to merge. Right now the `role arn` is optional and if you don't specify it will break.

Reply:
done. could you take a look.